system/cache/providers/CFProvider.cfc

```diff
@@ -212,10 +212,13 @@ component
      * Get a structure of all the keys in the cache with their appropriate metadata structures. This is used to build the reporting.[keyX->[metadataStructure]]
      */
     struct function getStoreMetadataReport(){
-		return getKeys()
-			.map( function( item ){
-				return getCachedObjectMetadata( item );
+		var result = {};
+		getKeys()
```

Comment:
hmm, what is the difference of using map() vs this approach? Map does exactly the same

Reply:
array().map() returns an array. This function is supposed to return a struct.

Reply:
Ohhhhhhhhh, ok, then let’s switch it up to a `reduce()` function then, so we can eat our cake 🍰

Reply:
OK. All set.